pycbc/io/live.py

```diff
@@ -339,41 +337,42 @@ def upload(self, fname, gracedb_server=None, testing=True,
                                                              '_snr.png')
             psd_series_plot_fname = snr_series_fname.replace('.hdf',
                                                              '_psd.png')
-            pylab.figure()
+            pl.figure()
+            ref_time = int(self.merger_time)
             for ifo in sorted(self.snr_series):
                 curr_snrs = self.snr_series[ifo]
                 curr_snrs.save(snr_series_fname, group='%s/snr' % ifo)
-                pylab.plot(curr_snrs.sample_times, abs(curr_snrs),
-                           c=ifo_color(ifo), label=ifo)
+                pl.plot(curr_snrs.sample_times - ref_time, abs(curr_snrs),
```

Comment:
@titodalcanton Just trying to understand what you've changed. Is this the only line that matter? There are other changes, but they all look like style choices.

Reply:
It is actually line 350. The change here just makes the plot's x-axis easier to read.